Complete File Set:

DNYoussef opened the pull request:
evolve_state.py → Classic approach
evolve_circuit.py → φ-scaled approach
fibonacci_anyon_braiding.py → Fibonacci anyon toy model topological_placeholders.py → Braid compiler placeholders & scaling references quantum_circuit.py → Houses StandardCircuit & PhiScaledCircuit quantum_state.py → Standard states + positivity projection create_experiment.py (optional) → Simple Streamlit-based control file. With this, you can run:

Classic: python simulations/scripts/evolve_state.py
φ-Scaled: python simulations/scripts/evolve_circuit.py
Topological: python simulations/scripts/fibonacci_anyon_braiding.py

DNYoussef committed:
Complete File Set:

evolve_state.py → Classic approach
evolve_circuit.py → φ-scaled approach
fibonacci_anyon_braiding.py → Fibonacci anyon toy model
topological_placeholders.py → Braid compiler placeholders & scaling references
quantum_circuit.py → Houses StandardCircuit & PhiScaledCircuit
quantum_state.py → Standard states + positivity projection
create_experiment.py (optional) → Simple Streamlit-based control file.
With this, you can run:

Classic: python simulations/scripts/evolve_state.py
φ-Scaled: python simulations/scripts/evolve_circuit.py
Topological: python simulations/scripts/fibonacci_anyon_braiding.py

christophergorexyz commented:
Did you ever go through https://learngitbranching.js.org/ ???

It looks like this duplicates work done in the branch I pushed last night, and targets `main` instead of that branch. There will likely be conflicts.

Branch names should be purposeful - "DavidBranches" doesn't mean anything. What are you *doing* in this branch? Why does it exist?

I know this is tedious, but it's what necessary to collaborate on software

DNYoussef committed:
In quantum_circuit.py:
Fixed Result initialization issues by using QuTiP's solvers (sesolve) directly
Replaced * operator with @ for matrix multiplication in Qobj operations
Added proper type hints for Qobj variables
Fixed H_eff initialization to ensure proper Qobj type
In fibonacci_anyon_braiding.py:
Replaced all * operators with @ for Qobj matrix multiplication operations
Fixed braiding operations to use proper matrix multiplication syntax
In quantum_state.py:
Fixed the sum operation in state_w to maintain Qobj type using reduce
Fixed matrix multiplication in positivity_projection using @ operator
In app.py:
Fixed Bell state creation by properly implementing (|00> + |11>)/sqrt(2)
Added missing state_one import from quantum_state module
